### PR TITLE
DBR-260 - Clean language files

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team, 

I have applied some minor changes in language files, removing unused entries and having a constant for Advanced Reports URL.

I have also found that the content of "activate trial" page in Spanish laked of the help link, so, I have added.

Could you review?

### CODE REVIEW

File: src/i18n/es.js [hunk 1]
```diff
@@ -92,31 +89,19 @@ export default {
       <p>
         Activa el periodo de prueba y accede a Reportes detallados sobre el comportamiento de los
         usuarios en tu Sitio Web o E-commerce. Descubre cuáles son las páginas más visitadas, cuántos
-        visitantes poseen un Email que Doppler ha identificado y cuántos no.
+        visitantes poseen un Email que Doppler ha identificado y cuántos no. ¿Necesitas ayuda?
+        <a target="_blank" href="${urlHelpAdvancedReports}" ${patchForBlank}>HELP</a>.
```
Comment (andresmoschini) [inline src/i18n/es.js]: Hi @carodipietro, I have added this text in Spanish on the page of trial activation.

I think that we have missed it before. 

image

What do you think?

File: src/i18n/es.js [hunk 2]
```diff
@@ -149,7 +134,7 @@ export default {
         Descubre cuáles son las páginas más visitadas de tu Sitio Web o E-commerce, cuántos
         visitantes poseen un Email que Doppler ha identificado y cuántos no. ¡Sigue el recorrido de
         los usuarios, detecta puntos de fuga y oportunidades de mejora! Si necesitas ayuda, presiona
```
Comment (GusBaamonde) [inline src/i18n/es.js]: Si necesitas ayuda, presiona Ayuda! Suena a Moria, si queres llorar llora :)

Comment (andresmoschini) [reply src/i18n/es.js]: jeje, así está en los demás.

Comment (GusBaamonde) [reply src/i18n/es.js]: Suena raro

File: src/i18n/es.js (hunk 1 above)
Comment (andresmoschini) [reply src/i18n/es.js]: Mergeo, si está mal, me avisan y revierto.

**Pull request merged**